### ISSUE TITLE
Compatibility of NVDA OCR with Python 3

### ISSUE THREAD

**Issue by lukaszgo1** [nvaccess/nvda-ocr#6]
For many people using older versions of Windows the OCR add-on is invaluable in day to day work. In some cases the Tesserac engine performs better than the build-in Windows 10 one, so even on Windows 10 add-on has some advantages. What are your plans regarding porting to Python 3? cc @feerrenrut as someone from NV Access and 
@jcsteh as an initial author.

**Comment by lukaszgo1** [comment]
After discussion with @nvaccessAuto I've decided to fork OCR add-on and maintain it fork the time being. For people who would like to use it with NVDA 2019.3 the compatible release would be available on NVDA add-ons page soon. Issues should be reported against https://github.com/lukaszgo1/nvda-ocr

As this issue is now solved I'm closing it.
